assessments/guide/README.md

```diff
@@ -0,0 +1,46 @@
+To provide the CNCF’s TOC with effective
+information about the security of different projects,
+this document outlines the procedure by which a project should be evaluated.
+
+## Roles
+
+* [project lead](project-lead.md)
+* [security reviewers](security-reviewer.md)
+
+## Steps
+
+1. Create tracking issue
+   * Issue may be a request from TOC liason or project itself
+2. Project provides self-assessment
+   * [project lead](project-lead.md) responds to the issue with draft document (see [outline](outline.md))
+   * issue assigned to lead [security reviewer](security-reviewer.md) who
+   will recruit a second reviewer, if one not already assigned, and facilitate
+   the process
+3. Inital review
+   * Upon request by the project, security reviewer may do an inital review to
+   verify completeness, ask for clarifications and provide quick feedback
+   * Project posts their document to the group mailing list, allowing at least
+   one week for review before presenting to the WG
+4. Presentation
+   * Project lead presents to WG
+   * This will be recorded as part of standard WG process
+5. Final assessment
+   * Reviewers provide final feedback with recommendations
+   * Either project lead or reviewers may request further WG discussion
+   * Project lead prepares a PR to /assessments/project-docs/project-name/
+
+## Additional Process Notes
+
+Iteration is expected; however, we expect quick turnaround (at most a week).
```

Comment:
Is there a desire to have an explicit followup process, ie an annual review?  I think the notion that a single point in time review ignores the reality that threats change over time. someone using this review process as evidence of "trustworthiness" can be lulled into thinking something is "safe" since it was reviewed N years ago.

Reply:
yes!  added as open issue: https://github.com/cncf/sig-security/issues/152  (so we can get this PR merged for initial reviews that are currently blocked, then add more detail on annual review process)